components/scream/src/share/grid/point_grid.cpp

```diff
@@ -5,30 +5,65 @@
 namespace scream {
 
```

Comment:
Likewise, I think the scorpio-interface tests will use a pointsgrid.  So one or the other PR will have to resolve this conflict, depending on which PR is merged first.